### DISCUSSION TITLE
Changing the logging level

### DISCUSSION THREAD

**Discussion by bernardtrigaux** [hassio-addons/app-tailscale#107] [Q&A]
Tailscale logging level : I recently installed tailscale addon. Really simple et effective.
I just observe a very verbose logging. How can I reduce this ?
2022/08/23 09:29:26 Accept: TCP{100.107.102.6:40200 > 100.xx.xx.xx:8123} 52 tcp non-syn
2022/08/23 09:29:26 magicsock: disco: node [QdaoF] d:999da76ac0fe3c62 now using 192.168.2.1:42799
2022/08/23 09:29:34 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn
2022/08/23 09:29:43 magicsock: disco: node [spSBG] d:4e2242017a4d5c48 now using 192.168.2.162:41641
2022/08/23 09:29:50 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn
2022/08/23 09:29:59 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn
2022/08/23 09:30:04 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn
2022/08/23 09:30:20 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn
2022/08/23 09:30:29 Accept: TCP{100.xx.xx.xx:55677 > 100.xx.xx.xx:8123} 40 tcp non-syn

**Comment by lmagyar** [comment]
Tailscaled is quite chatty, I made a PR #138 few weeks ago to make these messages configurable, you can test it in https://github.com/lmagyar/homeassistant-addon-tailscale or with additional support for the alpha Funnel feature in https://github.com/lmagyar/homeassistant-addon-tailscale-funnel.
